### ISSUE TITLE
PyPi does not like non-pypi dependencies in requirements

### ISSUE THREAD

**Issue by SpheMakh** [ratt-ru/CubiCal#287]
If you have a non-pypi dependency you get this exception:

```
ERROR: Packages installed from PyPI cannot depend on packages which are not
also hosted on PyPI.
```

Better to ask @sjperkins to make a release of montblac before pushing to PyPi

**Comment by SpheMakh** [comment]
This is line I'm referring to:
https://github.com/ratt-ru/CubiCal/blob/master/setup.py#L227

**Comment by bennahugo** [comment]
See https://github.com/ratt-ru/CubiCal/blob/master/.jenkins/1804.py3.docker

**Comment by SpheMakh** [comment]
> You can clone it and install or run pip install git+..... 

Yes, This is how I do it in the current base images of stimela. Was just letting ya'll know that it cannot be installed from PyPi in this form
